azure-pipelines.yml

```diff
@@ -11,7 +11,7 @@ pool:
 variables:
   DOCKER_BUILDKIT: 1
   k8sNamespace: 'dev-notification-client'
-  ciClusterServiceConnection: 'CI AKS'
```

Comment:
Vytvoril jsem novou service connection a nastavil ji podle daemona, kdyztak jukni jestli je to spravne.

Reply:
lgtm